api/src/main.ts

```diff
@@ -85,7 +85,7 @@ async function bootstrap() {
   }
 
   process.on('uncaughtException', (error) => {
-    if (error.stack.toLowerCase().includes('smtp'))
+    if (error.stack && error.stack.toLowerCase().includes('smtp'))
```

Comment:
```suggestion
    if (error.stack?.toLowerCase().includes('smtp'))
```
Hi @abdou6666, i think that we can use this shortest syntax